docs/devguide/flashfirmware.rst

```diff
@@ -1,16 +1,32 @@
 .. _flashfirmware:
 
-=================
-Flashing Firmware
-=================
+========================
+Build and flash firmware
+========================
 
-Building firmware
------------------
-Use yotta to build.
+micro:bit V1
+============
 
-Use target bbc-microbit-classic-gcc-nosd::
+This applies to MicroPython for the micro:bit V1, the source of which can be 
+found here: `bbcmicrobit/micropython <https://github.com/bbcmicrobit/micropython>`_.
 
-  yt target bbc-microbit-classic-gcc-nosd
+Dependencies
+------------
+- `cMake <https://cmake.org/>`_
+- `gcc <http://gcc.gnu.org/install/>`_
+- `git <https://git-scm.com/>`_
+- `ninja <https://ninja-build.org/>`_
+- `python <https://www.python.org/downloads/>`_
+- `yotta <http://docs.yottabuild.org//>`_
+
+The `yotta
+<http://docs.yottabuild.org//>`_ tool is used to build MicroPython, but before
+that takes place additional files have to be generated by the Makefile in
+preparation for the build, and additional data is added to the hex file after.
+
+Use yotta to build::
+
+  yt target bbc-microbit-classic-gcc@https://github.com/lancaster-university/yotta-target-bbc-microbit-classic-gcc
```

Comment:
This doesn't look right, I think it should be the `-nosd` target?
Did this come from other MicroPython instructions? In that case we'll need to update those as well?

Reply:
I think it was my assumption based on https://support.microbit.org/support/solutions/articles/19000123284-yotta-registry-deprecation I will update the target

Reply:
Done.